test.js

```diff
@@ -49,48 +50,58 @@ test('IPv4 HTTPS uses custom URLs', async t => {
 });
 
 test('IPv4 DNS timeout', async t => {
-	t.true(isIp.v4(await publicIp.v4({timeout: 2000})));
+	t.true(isIp.v4(await publicIpv4({timeout: 2000})));
 });
 
 test('IPv4 HTTPS timeout', async t => {
-	t.true(isIp.v4(await publicIp.v4({onlyHttps: true, timeout: 4000})));
+	t.true(isIp.v4(await publicIpv4({onlyHttps: true, timeout: 4000})));
 });
 
 test('IPv4 DNS cancellation', async t => {
 	const timeout = 5000;
-	const start = process.hrtime();
-	const promise = publicIp.v4({timeout});
+	const end = timeSpan();
+	const promise = publicIpv4({timeout});
 	promise.cancel();
 	await promise;
-	const difference = process.hrtime(start);
-	const milliseconds = ((difference[0] * 1e9) + difference[1]) / 1e6;
-	t.true(milliseconds < timeout);
+	t.true(end() < timeout);
 });
 
 test('IPv4 HTTPS cancellation', async t => {
 	const timeout = 5000;
-	const start = process.hrtime();
-	const promise = publicIp.v4({timeout, onlyHttps: true});
+	const end = timeSpan();
+	const promise = publicIpv4({timeout, onlyHttps: true});
 	promise.cancel();
 	await promise;
-	const difference = process.hrtime(start);
-	const milliseconds = ((difference[0] * 1e9) + difference[1]) / 1e6;
-	t.true(milliseconds < timeout);
+	t.true(end() < timeout);
 });
 
 // Impossible DNS timeouts seems unreliable to test on a working connection
 // because of caches, so we're only testing HTTPS.
 
 test('IPv4 HTTPS impossible timeout', async t => {
-	await t.throwsAsync(publicIp.v4({onlyHttps: true, timeout: 1}));
+	await t.throwsAsync(publicIpv4({onlyHttps: true, timeout: 1}));
 });
 
 if (!process.env.CI) {
 	test('IPv6 DNS', async t => {
-		t.true(isIp.v6(await publicIp.v6()));
+		t.true(isIp.v6(await publicIpv6()));
 	});
 
 	test('IPv6 HTTPS', async t => {
-		t.true(isIp.v6(await publicIp.v6({onlyHttps: true})));
+		t.true(isIp.v6(await publicIpv6({onlyHttps: true})));
 	});
 }
+
+test.serial('IPv4 or IPv6', async t => {
```

Comment:
If these were to run concurrently, other tests that stub modules so that they don't work would also affect these.